secretstores/local/env/envstore.go

```diff
@@ -77,3 +84,13 @@ func (s *envSecretStore) GetComponentMetadata() map[string]string {
 	metadata.GetMetadataInfoFromStructType(reflect.TypeOf(metadataStruct), &metadataInfo)
 	return metadataInfo
 }
+
+func (s *envSecretStore) isKeyAllowed(key string) bool {
+	switch key {
+	case "APP_API_TOKEN", "DAPR_API_TOKEN",
```

Comment:
I would really prefer if `dapr/kit` provided these reserved environment variables.

But for now this is fine.